atlasdb-config/src/main/java/com/palantir/atlasdb/http/FailoverFeignTarget.java

```diff
@@ -38,6 +38,8 @@
 import feign.Target;
 
 public class FailoverFeignTarget<T> implements Target<T>, Retryer {
+    public static final int DEFAULT_MAX_BACKOFF_MILLIS = 3000;
+
     private static final Logger log = LoggerFactory.getLogger(FailoverFeignTarget.class);
     private static final double GOLDEN_RATIO = (Math.sqrt(5) + 1.0) / 2.0;
```

Comment:
For consistency we normally have the logger as separate from everything else (I'm not sure why this wasn't originally in the target). Could we reorder these to

```
public class FFT<T> implements ... {
  psf Logger log = ...;

  psf int DMBM = 3000;
  (etc.)
}